client/components/App.jsx

```diff
@@ -4,15 +4,35 @@ class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      date: new Date()
+      date: new Date().toLocaleTimeString(),
+      id: props.id
     };
   }
+  // this.props.id = id
+
+  // hello this shaw /image desctiption text/image-number/ and that was an image 🐄
+
+  componentDidMount() {
+    this.intervalID = setInterval(
+      () => this.tick(),
+      1000
+    );
+  }
+  componentWillUnmount() {
+    clearInterval(this.intervalID);
+  }
+
+  tick() {
+    this.setState({
+      date: new Date().toLocaleString()
+    });
+  }
 
   render() {
```

Comment:
Join us on the dark side Nicolas.. deconstruction!
const { date } = this.state;
then on line 35 it would just be:
<h2>It is {date}.</h2>